libs/efa/src/lib/feature-journey-search/nearest-travel-point-search/nearest-travel-point-search.component.ts

```diff
@@ -54,5 +63,4 @@ export class NearestTravelPointListComponent implements OnInit {
     //   getFurtwangenFriedrichStreetThreeTravelPoint()
     // ]);
```

Comment:
Wenn ich es nicht besser wüsste würde ich sagen diese Klasse macht nichts mehr wenn dieser code auskommentiert ist oder ?

Reply:
Naja, die Komponente ist wieterhin die Container-Komponente für die Liste an TravelPoints. Die Liste wird sowohl für die Autocompletion als auch für die Suche näöchster Adressen/Stationen verwendet. Da der HTTP-Schnittstelle für die beiden Abfragen unterschieldich ist, macht es Sinn die HTTP-Aufrufe in eine übergelagerte Komponente auszulagern.